apps/src/blockly/addons/contextMenu.js

```diff
@@ -205,6 +205,69 @@ const registerKeyboardNavigation = function() {
   GoogleBlockly.ContextMenuRegistry.registry.register(keyboardNavigationOption);
 };
 
+/**
+ * Change workspace theme to classic CdoTheme
+ */
+const registerCdoTheme = function() {
+  const cdoThemeOption = {
+    displayText: function(scope) {
+      return (
+        (isCurrentTheme('classic', scope.workspace)
+          ? '✓ '
+          : `${msg.enable()} `) + msg.blocklyClassicTheme()
+      );
+    },
+    preconditionFn: function(scope) {
+      if (isMusicLabTheme(scope.workspace)) {
+        return 'hidden';
+      } else if (isCurrentTheme('classic', scope.workspace)) {
+        return 'disabled';
+      } else {
+        return 'enabled';
+      }
+    },
+    callback: function(scope) {
+      localStorage.setItem('blocklyTheme', 'classic');
```

Comment:
In Java Lab we store a `display_theme` variable on user preferences: [frontend api](https://github.com/code-dot-org/code-dot-org/blob/dd9b96b3a79cae3f32262dc1ceca8dfd8d878200/apps/src/lib/util/UserPreferences.js#L30). Can we reuse this variable? Or add to the object? I can see the themes diverging between blockly and Java Lab, and we would want to decide if we want the theme to persist across different labs.

Reply:
I'd be nervous to reuse a variable that's exclusively used today for Java Lab, but if you have suggestions for how to go about that, I'm certainly open to it.

It looks like Java Lab just has a "light" and "dark" themes? For Blockly, we're also expecting to add themes for high contrast and limited color vision. 

I wonder if it would make sense to work with localStorage for now until these themes are locked across Blockly labs?

Reply:
I think while we are working on this it's fine to keep in local storage, but we should eventually align things so all theming is done in one place. That could be that we change the displayTheme variable to be an object, something like `{'javalab': 'dark', 'blockly: 'dark'}`. We could also discuss if we want to have the same theme types for all labs, but that feels like painting ourselves into a corner.